e2e-tests/oauth-flow/rfc021/node-A/nginx.conf

```diff
@@ -1,18 +1,19 @@
+load_module /usr/lib/nginx/modules/ngx_http_js_module.so;
```

Comment:
Ah, it's not actually JavaScript; https://nginx.org/en/docs/njs/index.html

Or it is, but a custom implementation..?